scss/base.scss

```diff
@@ -34,5 +34,15 @@ mark {
     overflow-wrap: break-word;
 }
 
+code {
+    font-family: monaco, Consolas, "Lucida Console", monospace;
+    font-size: 0.8em; /* Slightly smaller font size */
```

Comment:
I think you should use `govuk-font-size` for this. See the type scale here https://design-system.service.gov.uk/styles/type-scale/